tests/functional/object/mcg/test_lifecycle_configuration.py

```diff
@@ -0,0 +1,365 @@
+        3. Create a multipart upload for the bucket
+        4. Upload a few parts
+        5. Manually expire the multipart-upload
+        6. Wait for the multipart-upload to expire
+        7. Verify that the parts were deleted at the noobaa-db
+        """
+        parts_amount = 5
+        key = "test_obj_123"
+        origin_dir = test_directory_setup.origin_dir
+        res_dir = test_directory_setup.result_dir
+
+        # 1. Create a bucket
+        bucket = bucket_factory(interface="OC")[0].name
+
+        # 2. Set lifecycle configuration
+        lifecycle_policy = LifecyclePolicy(
+            AbortIncompleteMultipartUploadRule(days_after_initiation=1)
+        )
+        mcg_obj.s3_client.put_bucket_lifecycle_configuration(
+            Bucket=bucket, LifecycleConfiguration=lifecycle_policy.as_dict()
+        )
+        logger.info(
+            f"Sleeping for {PROP_SLEEP_TIME} seconds to let the policy propagate"
+        )
+        sleep(PROP_SLEEP_TIME)
+
+        # 3. Create a multipart upload
+        upload_id = create_multipart_upload(mcg_obj, bucket, key)
+
+        # 4. Upload a few parts
+        awscli_pod.exec_cmd_on_pod(
+            f'sh -c "dd if=/dev/urandom of={origin_dir}/{key} bs=1MB count={parts_amount}; '
+            f'split -b 1m  {origin_dir}/{key} {res_dir}/part"'
+        )
+        parts = awscli_pod.exec_cmd_on_pod(f'sh -c "ls -1 {res_dir}"').split()
+        upload_parts(
+            mcg_obj,
+            awscli_pod,
+            bucket,
+            key,
+            res_dir,
+            upload_id,
+            parts,
+        )
+
+        # 5. Manually expire the parts and the multipart-upload
+        expire_multipart_upload(upload_id)
+
+        # 6. Wait for the parts and multipart-upload to expire
+        for http_response in TimeoutSampler(
+            timeout=TIMEOUT_THRESHOLD,
+            sleep=TIMEOUT_SLEEP_DURATION,
+            func=list_multipart_upload,
+            s3_obj=mcg_obj,
+            bucketname=bucket,
+        ):
+            if "Uploads" not in http_response or len(http_response["Uploads"]) == 0:
+                break
+            logger.warning(f"Upload has not expired yet: \n{http_response}")
+
+        # 7. Verify that the parts were deleted at the noobaa-db
+        bucket_id = exec_nb_db_query(
+            f"SELECT _id FROM buckets WHERE data->>'name' = '{bucket}'",
+        )[0].strip()
+        multiparts_md_list = exec_nb_db_query(
+            f"SELECT data FROM objectmultiparts WHERE data->>'bucket' = '{bucket_id}'"
+        )
+        for md_string in multiparts_md_list:
+            md = json.loads(md_string)
+            assert (
+                "deleted" in md
+            ), f"Multipart upload was not expired as expected: {md}"
+
+    @tier1
+    @pytest.mark.polarion_id("OCS-6559")
+    def test_noncurrent_version_expiration(self, mcg_obj, bucket_factory, awscli_pod):
+        """
```

Comment:
Add args in docstring

Reply:
We typically don't include fixture args in docstrings for test functions, so I think it's best to keep it consistent with the rest of the project.